### DISCUSSION TITLE
Where is `console::time_stamp_with_label(str)` ? should I open an issue for this?

### DISCUSSION THREAD

**Discussion by maximetinu** [wasm-bindgen/wasm-bindgen#3775] [Q&A]
The title says it all

I was expecting to see a `fn time_stamp_with_label()` function [here](https://github.com/rustwasm/wasm-bindgen/blob/adcf7786d14943244cbe418a2a17d00430008e2e/crates/web-sys/src/features/gen_console.rs#L1421), but `web_sys` only exposes it without labels, contrary to the specification: https://developer.mozilla.org/en-US/docs/Web/API/console/timestamp_static

**Comment by daxpedda** [comment]
Most likely because these are non-standard APIs, as pointed out in the documentation you posted.

On a side-note: I'm unfortunately very unfamiliar with this API, but would [`Performance.mark()`](https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark) do the trick for you as well?
